### PR TITLE
[MBL-18104][Student][Teacher] Updated Heap

### PR DESCRIPTION
**Pull request by tamaskozmer**
Test plan: Verify that Heap tracking works correctly.

refs: MBL-18104
affects: Student, Teacher
release note: none

## Checklist

- [x] Tested in light mode

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

File: README.md [hunk 1]
```diff
@@ -59,7 +59,6 @@ App | Description
 Module | Description
    --- | ---
 annotations     | A wrapper for the PSPDFKit library and logic for annotation handling and converting in PDF documents.
-blueprint       | An MVP Architecture that depends on PandaRecyclerView. (deprecated)
```
Comment (hermannakos) [inline README.md:62]: Sneaky

**Review by hermannakos — APPROVED** [review]

**Review by kristofnemere — APPROVED** [review]
QA+1

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**